dll/win32/msgina/prototype/prototype.c

```diff
@@ -0,0 +1,196 @@
+// prototype.cpp : définit le point d'entrée pour l'application.
```

Comment:
Are you going to remove the prototype again?

Reply:
Prototype is a temporary thing for me to quickly test/code the functionality, and once it's done, I plug it in msgina/shell32/syssetup , and remove the prototype, of course. (And clean up the commit history ^^)